symengine/subs.h

```diff
@@ -32,15 +32,15 @@ class SubsVisitor : public BaseVisitor<SubsVisitor>
         SymEngine::umap_basic_num d;
         RCP<const Number> coef;
 
-        auto it = subs_dict_.find(x.coef_);
+        auto it = subs_dict_.find(x.get_coef());
         if (it != subs_dict_.end()) {
             coef = zero;
             Add::coef_dict_add_term(outArg(coef), d, one, it->second);
         } else {
-            coef = x.coef_;
+            coef = x.get_coef();
```

Comment:
Here is an example, where the result of `get_coef()` is saved. What this means is that the refcount is *not* incremented in this case. Which is a bug, because if `x` goes out of scope before `coef`, this will segfault. That is my understanding --- do you agree with me, or am I wrong? If I am right, then this is inherently unsafe, and thus this usage should not be used.